.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+rvm:
+  - 2.0
+  - 2.1
+sudo: false
+cache: bundler
+script:
+  - bundle exec rake test rubocop
```

Comment:
Because travis was running only "test" and not "rubocop" (at least there's not trace of it in the console).

I'll fix the rake default target to execute rubocop as well.